src/record/list.js

```diff
@@ -209,4 +217,27 @@ List.prototype._applyUpdate = function( message ) {
 	Record.prototype._applyUpdate.call( this._record, message );
 };
 
-module.exports = List;
\ No newline at end of file
+/**
+ * Validates that the index provided is within the current set of entries.
+ *
+ * @param {Number} index
+ *
+ * @private
+ * @returns {Number}
+ */
+List.prototype._hasIndex = function( index ) {
+	var hasIndex = false;
+	var entries = this.getEntries();
+	if( typeof index !== 'undefined' ) {
```

Comment:
Very minor, but if( index !== undefined) is a bit quicker since it doesn't create two strings